### DISCUSSION TITLE
这个怎么用来训练表格结构识别？

### DISCUSSION THREAD

**Discussion by nissansz** [OleehyO/TexTeller#43] [Q&A]
这个怎么用来训练表格结构识别？

**Comment by OleehyO** [comment]
只要把表格用形式语言进行描述，然后构造足够多的训练数据就能训练，跟公式识别没有太大区别。

**Comment by nissansz** [comment]
我有paddle格式label，不知道怎么用你这个代码训练，推理

**Reply by OleehyO** [reply]
可以参考[这个目录](https://github.com/OleehyO/TexTeller/tree/pre_release/src/models/ocr_model/train/dataset)中数据的格式，然后重新构造词典即可。
推理的代码在[这里](https://github.com/OleehyO/TexTeller/blob/pre_release/src/inference.py)
